styles.css

```diff
@@ -0,0 +1,91 @@
+body {
+  width: 375px;
+  padding: 0px;
+  margin: 0px;
+}
+
+header {
+  top: 0px;
```

Comment:
Currently, this CSS property has no use in your code. Check this [link](https://developer.mozilla.org/en-US/docs/Web/CSS/position) for more information on how to use it.
Always remember to keep your codebase DRY and CLEAN from nice codes with no contribution to your purpose.